### PR TITLE
Format Selects Minor Improvements

### PR DESCRIPTION
**Pull request by twiss**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -368,15 +368,16 @@ <h1 id="fileName" class="no-zen"><span id="currentFileLocation" style="display:n
 					</li>
 					<li>
 						<select data-change="style" id="style-select" data-l10n-title="formatting">
-							<option value="" disabled selected data-l10n-id="format-block"></option>
-							<option value="p" data-l10n-id="paragraph"></option>
+							<option value="" disabled selected data-l10n-id="format-block"			style="font-size: 1.8rem;	font-weight: initial;	color: initial;"></option>
+							<option value="p" data-l10n-id="paragraph"								style="font-size: 1.8rem;	font-weight: initial;	color: initial;"></option>
```
Comment (ferndot) [inline index.html:372]: Extra tabs in here

File: index.html [hunk 2]
```diff
@@ -368,15 +368,16 @@ <h1 id="fileName" class="no-zen"><span id="currentFileLocation" style="display:n
...
-							<option value="h6" data-l10n-id="heading-n" data-l10n-args='{ "n": 6 }' style="text-decoration: underline; color: #444;"></option>
+							<!-- `font-size`s are multiplied by 1.8rem -->
+							<option value="h1" data-l10n-id="heading-n" data-l10n-args='{ "n": 1 }' style="font-size: 2.7rem;	font-weight: bold;		color: initial;"></option>
+							<option value="h2" data-l10n-id="heading-n" data-l10n-args='{ "n": 2 }' style="font-size: 2.106rem;	font-weight: bold;		color: initial;"></option>
+							<option value="h3" data-l10n-id="heading-n" data-l10n-args='{ "n": 3 }' style="font-size: 1.8rem;	font-weight: bold;		color: initial;"></option>
+							<option value="h4" data-l10n-id="heading-n" data-l10n-args='{ "n": 4 }' style="font-size: 1.8rem;	font-weight: initial;	color: initial;	text-decoration: underline;"></option>
+							<option value="h5" data-l10n-id="heading-n" data-l10n-args='{ "n": 5 }' style="font-size: 1.8rem;	font-weight: bold;		color: #555;"></option>
+							<option value="h6" data-l10n-id="heading-n" data-l10n-args='{ "n": 6 }' style="font-size: 1.8rem;	font-weight: initial;	color: #444;	text-decoration: underline;"></option>
```
Comment (ferndot) [inline index.html:380]: Tabs here too

File: index.html (hunk 1 above)
Comment (twiss) [inline index.html:372]: If you set your tab width to 4 it's aligned. I could changes it to spaces if you want, but that makes things a bit harder.

Comment (ferndot) [inline index.html:372]: Oh, it's for readability? Carry on :)

**Pull request merged**